src/browser/getTokensStorage.ts

```diff
@@ -0,0 +1,8 @@
+import { JwtTokensInterface } from '..'
+
+declare const localStorage;
```

Comment:
Do we want to remove the browser directory?

Reply:
I think it has to be there as we might need to reuse some part (now that solc-js is not there the library can probably be run on the browser too)